docs/content/how-to/aws/index.md

```diff
@@ -0,0 +1,555 @@
+## Architecture and Workflow
+
+In order to deploy a HostedCluster you have two options, using the **CLI** and using **Openshift/Kubernetes objects (CRs)**, like *HostedCluster* and *NodePool* which are the top level objects that we will focus on. From both ones the proper Controller for that resource will take care of the object generation under that CR.
+
+This is how looks like when you have the Hypershift Operator deployed into the Openshift Management Cluster
+
+```mermaid
+flowchart LR
+    style HO fill:#BAE0E9,stroke:#333,stroke-width:3px
+    style HNDEP fill:#BAE0E9,stroke:#333,stroke-width:3px
+
+    subgraph "Openshift Management Cluster"
+        subgraph HN[Hypershift Namespace]
+        HO[fa:fa-server  Hypershift-Operator] --> HNDEP[Deployments];
+            subgraph HNDEP[Deployments]
+            direction LR
+                External-DNS --> descExD{{Manages Provider DNS *optional* }};
+                Operator --> descOp{{Manages Hosted Clusters}};
+            end
+        end
+    end
+```
+> Legend:
+>
+> - Cyan: Provider agnostic OCP objects.
+
+The CLI will create 2 deployments:
+
+- **ExternalDNS**: It will take care of the DNS management with the Cloud Provider. It will try to create entries into the domain/subdomain set in the install command.
+- **Operator**: Stands for Hypershift Operator and it has some controllers inside which manages the HostedClusters and NodePools created in the cluster's namespace.
+
+> **NOTE:** Even using External DNS feature or not, as we said before, we will need to have an already registered DNS domain (or at least delegated) in *Route 53*. There, the Public entries will be created to allow access from outside.
+
+
+### HostedCluster
+
+This is the Hosted Cluster namespace where you as a user will create the definition of your cluster and your nodes. An easy way to create a hosted cluster it's just using the CLI as we've seen in the last sample.
+
+HostedCluster and NodePool controllers (part of Hypershift Operator) are monitoring the HostedCluster and NodePool objects present o created in the whole cluster. So once you create one of these components, the controllers will begin the reconciliation.
+
+You will need to have in mind that some more objects like Secrets and ConfigMaps (described in the diagram) are needed to achieve a succesfully HostedCluster deployment.
+
+From these mentioned objects, the Hypershift operator will create some more in 2 places, Openshift infrastructure and also in AWS Cloud Provider. Let's take a look to the following diagram to discover which objects are created in Openshift.
+
+```mermaid
+flowchart TB
+    style HO fill:#BAE0E9,stroke:#333,stroke-width:3px
+    style SC2 fill:#BAE0E9,stroke:#333,stroke-width:3px
+    style CMC2 fill:#BAE0E9,stroke:#333,stroke-width:3px
+    style NPO fill:#D3FFCE,stroke:#333,stroke-width:3px
+    style HCO fill:#D3FFCE,stroke:#333,stroke-width:3px
+
+    AD[fa:fa-user Administrator]
```

Comment:
Let's use the personas: https://hypershift-docs.netlify.app/reference/concepts-and-personas/ for consistency